src/ci/docker-compose/cromwell-test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM us.gcr.io/broad-dsp-gcr-public/base/jre:8-debian
+FROM adoptopenjdk:8
```

Comment:
Before the recent change this had been `openjdk:8`; is there a reason behind this new choice of value?

Reply:
I just saw that in some other file Chris recently changed `openjdk:8` to `adoptopenjdk:8`. I think this is something Trivy-related, but I don't know enough context around this. @cjllanwarne should we use  `openjdk:8` or `adoptopenjdk:8` here?

Reply:
Confirmed here: https://broadinstitute.slack.com/archives/GHYJZ2ZE0/p1611084632053600